brave/pom.xml

```diff
@@ -136,7 +126,11 @@
       <id>release</id>
       <properties>
         <!-- The brave jar needs to be Java 1.6 bytecode -->
-        <main.java.version>1.6</main.java.version>
+        <maven.compiler.source>1.6</maven.compiler.source>
+        <maven.compiler.target>1.6</maven.compiler.target>
+        <!-- Clear the release tag, so that we can compile Platform without
```

Comment:
example of clearing the tag.. this is what broke the bytecode version by accident last release, as the source/target were overridden by a hard-coded release property. Setting this to 6 breaks our ability to compile `brave.internal.Platform`, so clearing is the only way.